src/advanced/bitcoin-anchoring.md

```diff
@@ -221,8 +221,8 @@ parameters](../architecture/configuration.md) to be set.
 
 #### Bitcoind node
 
-The service uses third-party bitcoin node to communicate with the
-bitcoin blockchain network. As for Exonum v 0.1, [Bitcoin
+The service uses a third-party Bitcoin node to communicate with the
+Bitcoin blockchain network. As for Exonum v 0.1, [Bitcoin
```

Comment:
Slightly off-topic: propose to replace "As for Exonum v 0.1" with "Currently" in order for the article to look fresher.